extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/CodeAuthenticationMechanism.java

```diff
@@ -162,7 +177,10 @@ private static QuarkusSecurityIdentity augmentIdentity(SecurityIdentity security
         }
 
         params.put("code", code);
-        params.put("redirect_uri", buildCodeRedirectUri(context, absoluteUri, getDynamicPath(context, absoluteUri)));
+        String redirectPath = getRedirectPath(context, absoluteUri);
+        String redirectUriParam = buildRedirectUri(context, absoluteUri, redirectPath);
+        LOG.debug("Token request redirect_uri parameter: " + redirectUriParam);
```

Comment:
@sberyozkin I let you change that one to use `debugf`. Ping me when it's done. Thanks!

Reply:
@gsmet Hi Guillaume, it is done, there were 2 more instances of `debug` in some other parts of this class. Thanks for spotting it.